lib/buffy/throttle_and_timed.ex

```diff
@@ -291,16 +334,43 @@ defmodule Buffy.ThrottleAndTimed do
       @doc """
       Function that returns a key from incoming args.
 
-      Defaults to `args |> :erlang.term_to_binary() |> :erlang.phash2()`
+      Defaults to using `:erlang.term_to_binary/1` and  `:erlang.phash2()`
+      but can be overwritten for more efficient memory usage.
       """
-      @spec args_to_key(any()) :: non_neg_integer()
-      def args_to_key(args), do: args |> :erlang.term_to_binary() |> :erlang.phash2()
+      @impl Buffy.ThrottleAndTimed
+      @spec args_to_key(Buffy.ThrottleAndTimed.args()) :: non_neg_integer()
+      def args_to_key(args) do
+        args
+        |> :erlang.term_to_binary()
+        |> :erlang.phash2()
+      end
+
       defoverridable args_to_key: 1
 
       defp key_to_name(key) do
         {:via, unquote(registry_module), {unquote(registry_name), {__MODULE__, key}}}
       end
 
+      @doc """
+      Returns the amount of jitter in milliseconds to add to the throttle time.
```

Comment:
NIT on the use of `jitter` in the doctstring - 

Although this makes sense `loop_interval` is a more generic gap until the next handle_throttle being fired, and jitter sounds like a specialized form of a gap until the next trigger. 

Jitter sounds like it can randomly change and is an additional time delay, whereas a loop interval is geared more towards time given to fire repetitively. Although you can override this function to randomly change that interval (like what a jitter sounds like it can do), I guess the concept of a loop interval is more generic than that.

Reply:
Yep, you are correct. That was a copy paste fail 🤦